starter_code/js/app.js

```diff
@@ -0,0 +1,25 @@
+$( document ).ready(function(e) {
+
+	$('.readmore').click(function(e){
+		$('.readmore').hide();
+		$('#show-this-on-click').slideDown();
+		$('.readless').removeClass('hide');
+		e.preventDefault();
+	});
+
+	$('.readless').click(function(e){
+		$('#show-this-on-click').slideUp();
+		$('.readless').addClass('hide');
+		$('.readmore').show();
+		e.preventDefault();
+	});
+
+	$('.learnmore').click(function(e){
+		$('#learnmoretext').slideDown();
+		$('.learnmore').hide();
+		e.preventDefault();
+	});
+
+e.preventDefault();
```

Comment:
The purpose of the ```preventDefault``` in this exercise is to prevent the anchor from changing your scroll position when you click them.  There's no need to prevent default of $.ready, I don't think there are any default behaviours to this.